broker/tasks/alb.py

```diff
@@ -25,6 +31,68 @@ def get_lowest_used_alb(listener_arns):
     listener_data = listener_data["Listeners"][0]
     return listener_data["LoadBalancerArn"], listener_data["ListenerArn"]
 
+
+def load_albs(dedicated_listener_arns):
+    stmt = insert(DedicatedALBListener).values(
+        [dict(listener_arn=arn) for arn in dedicated_listener_arns]
+    )
+    stmt = stmt.on_conflict_do_nothing(index_elements=["listener_arn"])
+    db.session.execute(stmt)
+
+
+def get_lowest_dedicated_alb(service_instance, db):
+    # TODO: handle grabbing the next available ALB when ours are full, or nearly so
```

Comment:
How are the TODOs going to be handled? Follow up PRs?

Reply:
yeah - that's the plan (although I'm working on that today, so it might end up just ballooning this PR instead)

Reply:
I might prefer separate follow up PRs if possible. It's much easier to review the code in more digestible chunks